src/backend/updates.rs

```diff
@@ -79,7 +79,17 @@ fn post_upgrade() {
 }
 
 #[allow(clippy::all)]
-fn sync_post_upgrade_fixtures() {}
+fn sync_post_upgrade_fixtures() {
+    mutate(|state| {
+        // Compensate for the executed by failed proposal #/post/1159871
+        state
```

Comment:
deploying this locally resulted in error, had to comment it out, probably same will happen on staging

Reply:
Very good point, thanks! Will make it fail-safe.